tests/integration/utils/flow_tools.py

```diff
@@ -93,17 +94,28 @@ def __make_scheduler(reg: str, **opts: Any) -> Scheduler:
         schd.workflow_db_mgr.on_workflow_shutdown()
 
 
+async def _install_without_symlinks(schd):
```

Comment:
It may become necessary in the future to allow a pytest marking to skip this, but for now it's not required.